ncoreparser/parser.py

```diff
@@ -32,17 +33,25 @@ def get_items(self, data):
         sizes = self.size_pattern.findall(data)
         seed = self.seeders_pattern.findall(data)
         leech = self.leechers_pattern.findall(data)
+        poster_image = self.poster_image_pattern.findall(data)
+
+        # There can be torrents without poster_image
+        if (len(poster_image) < len(leech)):
+            for i in range(len(poster_image), len(leech)):
```

Comment:
For example: if we have 5 leech, the poster is missing in the first and last elements. This  loop will set no cover for the 3. 4. and 5. element what is not correct.